udp-native/src/main/java/org/ballerinalang/stdlib/udp/endpoint/ConnectionlessClientActions.java

```diff
@@ -80,18 +80,18 @@ public static Object initEndpoint(Environment env, BObject client, BMap<BString,
             selectorManager.start();
         } catch (SelectorInitializeException e) {
             log.error(e.getMessage(), e);
-            balFuture.complete(SocketUtils.createSocketError("unable to initialize the selector"));
+            balFuture.complete(SocketUtils.createSocketError("Unable to initialize the selector."));
             return null;
         } catch (SocketException e) {
-            balFuture.complete(SocketUtils.createSocketError("unable to bind the local udp port"));
+            balFuture.complete(SocketUtils.createSocketError("Unable to a port."));
```

Comment:
"Unable to a port." - Shall we make make this more expressive?

Reply:
fixed